vault/seal_manager.go

```diff
@@ -200,3 +240,77 @@ func (sm *SealManager) InitializeBarrier(ctx context.Context, ns *namespace.Name
 
 	return nsSealKeyShares, nil
 }
+
+type SimpleBarrier interface {
```

Comment:
Don't know about the name here... It can wrap barrier, but could also wrap a physical backend...